### PR TITLE
23 name change

### PR DESCRIPTION
**Pull request by retrevor**
According to https://github.com/MetaProvide/nextcloud-swarm-plugin/issues/23

### CODE REVIEW

File: lib/AppInfo/Application.php [hunk 1]
```diff
@@ -76,7 +76,10 @@ public function registerEventsScripts(IEventDispatcher $dispatcher) {
 		// 	\OCP\Util::addScript(self::APP_ID, 'dist/collaboration');
 		// });
 		//\OCP\Util::addScript(self::APP_ID, 'beeswarmfilelist' );
+<<<<<<< HEAD
```
Comment (mwalbeck) [inline lib/AppInfo/Application.php:79]: There is a merge conflict here that wasn't handled properly

File: lib/Settings/Admin.php [hunk 2]
```diff
@@ -56,8 +56,13 @@ public function getForm(): TemplateResponse {
 		$mounts = json_decode($mounts_json,  true);
 
 		// Get all valid Beeswarm storages
+<<<<<<< HEAD
```
Comment (mwalbeck) [inline lib/Settings/Admin.php:59]: Also a merge conflict here

**Review by mwalbeck — CHANGES_REQUESTED** [review]
There are some unhandled merge conflicts but the rest looks good.

File: lib/AppInfo/Application.php (hunk 1 above)
Comment (retrevor) [reply lib/AppInfo/Application.php]: Corrected

File: lib/Settings/Admin.php (hunk 2 above)
Comment (retrevor) [reply lib/Settings/Admin.php]: Corrected

**Review by retrevor — COMMENTED** [review]
Reviewed and corrected

File: lib/AppInfo/Application.php [hunk 3]
```diff
@@ -76,10 +76,7 @@ public function registerEventsScripts(IEventDispatcher $dispatcher) {
 		// 	\OCP\Util::addScript(self::APP_ID, 'dist/collaboration');
 		// });
 		//\OCP\Util::addScript(self::APP_ID, 'beeswarmfilelist' );
-<<<<<<< HEAD
-=======
-		\OCP\Util::addScript(self::APP_ID, "main");
->>>>>>> f2e5dee91ff6d550b63b77fbb990b8720d52fe99
+		//\OCP\Util::addScript(self::APP_ID, "main");
```
Comment (mwalbeck) [inline lib/AppInfo/Application.php:79]: New question, why is everything in this function commented

Comment (retrevor) [reply lib/AppInfo/Application.php]: Yes, it was due to tests. At this moment, there is no need to inject js scripts

**Review by mwalbeck — APPROVED** [review]

**Pull request merged**